Sources/App/Controllers/API/API+PackageController+GetRoute.swift

```diff
@@ -49,6 +45,7 @@ extension API.PackageController {
             async let buildInfo = API.PackageController.BuildInfo.query(on: database,
                                                                         owner: owner,
                                                                         repository: repository)
+            async let forkedFromInfo = forkedFromInfo(on: database, fork: packageResult.repository.forkedFrom)
```

Comment:
We're potentially running another query, using `async let` makes this concurrent.